parcels/particle.py

```diff
@@ -77,6 +77,7 @@ def __init__(self, lon, lat, grid, cptr=None):
         self.lat = lat
         self.xi = np.where(self.lon >= grid.U.lon)[0][-1]
         self.yi = np.where(self.lat >= grid.U.lat)[0][-1]
+        self.active = 1
```

Comment:
C traditionally represents booleans as integers and this way the interfacing with the JIT backend seemed easiest. That said, I haven't tried interfacing through native booleans yet.